app/block/[block]/page.tsx

```diff
@@ -95,12 +95,12 @@ export default async function BlockDetailsPage({
 
   if (!block || error || !teams) notFound()
 
-  const { timestamp, block_number, gas_used, proofs, hash } = block
+  const { timestamp, block_number, gas_used, proofs: blockProofs, hash } = block
 
-  const proofsWithTeams = proofs.map((proof) => {
+  const proofs = blockProofs.map((proof) => {
     const team = teams.find((t) => t.user_id === proof.user_id)
     return { ...proof, team }
-  })
+  }) as Proof[]
```

Comment:
Why are we overriding the inferred type?

Reply:
Oops thanks, good catch... It was the new `size` type which needed to be `size?: number | null`